lightspeed/assemblies/assembly_managing-admin-dashboard-telemetry.adoc

```diff
@@ -0,0 +1,48 @@
+ifdef::context[:parent-context: {context}]
+
+:_content-type: ASSEMBLY
+
+[id="managing-admin-dashboard-telemetry_{context}"]
+
+= Viewing and managing Admin dashboard telemetry
+
+:context: managing-admin-dashboard-telemetry
+
+[role="_abstract"]
+
+{LightspeedshortName} collects the following telemetry data by default:
+
+* *Operational telemetry data*
++
+This is the data that is required to operate and troubleshoot the Ansible Lightspeed service. For more information, please refer the Enterprise Agreement. You cannot disable the collection of operational telemetry data. 
```

Comment:
Just a suggestion to remove "please" here as the IBM SG recommends against it. 

"Do not use in IBM technical information or in instructions. Terms of politeness are superfluous, convey the wrong tone for technical material, and are not regarded the same way in all cultures. In marketing information, terms of politeness might be appropriate."

Reply:
Good catch, thank you. I've made the edit.